### PR TITLE
Crm457 974 - fixes

### PR DESCRIPTION
**Pull request by ivanELEC**
## Description of change

 [CRM4 Provider - Further information Bug Fixes](https://dsdmoj.atlassian.net/browse/CRM457-974)

Addressing these comments: [Comment 1](https://dsdmoj.atlassian.net/browse/CRM457-974?focusedCommentId=452018), [Comment 2](https://dsdmoj.atlassian.net/browse/CRM457-974?focusedCommentId=451280)

-  Update "Re-submitted" to "Resubmitted"
- Update badge colour used for Submitted and Resubmitted
- Fix incorrect styling for summary cards on Check Answers page

### CODE REVIEW

File: app/presenters/prior_authority/check_answers/base.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def title(**)
 
       def rows
         row_data.map do |row|
-          row_content(row[:head_key], row[:text], row[:actions], row[:head_opts] || {})
+          row_content(row[:head_key], row[:text], row[:head_opts] || {}, row[:actions] || nil)
```
Comment (jsugarman) [inline app/presenters/prior_authority/check_answers/base.rb:26]: is the row[:actions] `|| nil` bit needed? won't it be nil if not supplied anyway

```suggestion
          row_content(row[:head_key], row[:text], row[:head_opts] || {}, row[:actions])
```

**Review by jsugarman — COMMENTED** [review]
looks ok, but previous defaulting to an empty array for `actions` allows for extensibility and allows you to not supply an array as in
```ruby
govuk_summary_card(title: section.title, actions: (show_actions ? section.title_action : [])) do |card|
```

```ruby
#ufn_card

def title_action
  []
end
```

File: app/views/prior_authority/steps/check_answers/edit.html.erb [hunk 2]
```diff
@@ -22,6 +22,7 @@
       <% end %>
     <% end %>
 
+
```
Comment (jsugarman) [inline app/views/prior_authority/steps/check_answers/edit.html.erb:25]: **nit** unnecessary line break?

**Comment by ivanELEC** [comment]
a

> looks ok, but previous defaulting to an empty array for `actions` allows for extensibility and allows you to not supply an array as in
> 
> ```ruby
> govuk_summary_card(title: section.title, actions: (show_actions ? section.title_action : [])) do |card|
> ```
> 
> ```ruby
> #ufn_card
> 
> def title_action
>   []
> end
> ```

Yeah I've reverted those changes now but gone with keeping the method name semantics as I was confused with how the method was working

**Comment by ivanELEC** [comment]
in this case I still needed to keep the logic in for the different view on this report if it's being used on a submitted claim vs a sent back claim

**Review by jsugarman — APPROVED** [review]

**Pull request merged**